object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,22 @@
+/* exported omit */
+
+// We need a storage for the new object.
+// Store each property value pair of source.
+// Delete properties that are included in the keys array.
+// Return the storage.
+
+function omit(source, keys) {
+  var newObject = {};
+  for (var property in source) {
+    var value = source[property];
+    newObject[property] = value;
+    for (var i = 0; i < keys.length; i++) {
+      delete newObject[keys[i]];
+    }
```

Comment:
This is still inside the `for in` loop, try moving it below the `for in` loop and see if it still works 🤞